srcpkgs/xwinwrap/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'xwinwrap'
+pkgname=xwinwrap
+version=0.9
+revision=1
+build_style=gnu-makefile
+makedepends="libXrender-devel"
+short_desc="X11 Window as background"
+maintainer="SolitudeSF <solitudesf@protonmail.com>"
+license="MIT"
+homepage="https://github.com/Aaahh/xwinwrap"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=0e840586deaf6780d426dc49f7a344366198aead96535cc1eb2ac7ce366777df
+
+pre_install() {
+	vsed -i Makefile -e 's|/local||'
```

Comment:
Note that with the recent changes by @Gottox you can now use vsed with mostly the same semantics as sed

so `vsed -i <regex> <file>` works.